Alignment/LaserAlignment/python/LaserSeedGenerator_cff.py

```diff
@@ -3,7 +3,6 @@
 # stripCPE
 from RecoLocalTracker.SiStripRecHitConverter.StripCPEfromTrackAngle_cfi import *
 # pixelCPE
```

Comment:
Please also remove the corresponding comment line "`# pixelCPE`" (16 occurencies in this PR)